scripts/my-info.sh

```diff
@@ -92,13 +92,16 @@ function checkfiles() {
     mlog "Permission checks..."
     cat << EOF > /tmp/my-list.$$
 ${MYCROFT_HOME}
-${MYCROFT_HOME}/scripts/logs
+/var/log/mycroft/skills.log
+/var/log/mycroft/voice.log
+/var/log/mycroft/bus.log
+/var/log/mycroft/audio.log
```

Comment:
Is it worth dynamically populating this list of log files so that any future log files are picked up automatically? 

If not, we should probably at least add enclosure.log

Reply:
Good suggestion, I've replaced this with a function to find all files in the log folder.